### PR TITLE
[Survival] Caltrops cooldown update

### PR DESCRIPTION
**Pull request by Pewtro**
i am bad at numbers apparently. reverting my fix that wasn't a fix on this.

### CODE REVIEW

File: src/Parser/Hunter/Survival/Modules/Abilities.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.CALTROPS_TALENT,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: 15,
+        cooldown: 12,
```
Comment (sMteX) [inline src/Parser/Hunter/Survival/Modules/Abilities.js]: I may be wrong but how about this? http://www.wowhead.com/spell=194277/caltrops 
Shows cooldown of 15 seconds? 🤔

Comment (Pewtro) [reply src/Parser/Hunter/Survival/Modules/Abilities.js]: That is what prompted my fix - but then I was in-game testing something and noticed
image

Comment (sMteX) [reply src/Parser/Hunter/Survival/Modules/Abilities.js]: Any leggos or other talents that can affect this?

Comment (Pewtro) [reply src/Parser/Hunter/Survival/Modules/Abilities.js]: I found it 🤔 updating this and some others post raid, so just leave it up for now 😨

**Comment by Pewtro** [comment]
Already there we go 😄

**Pull request merged**